src/main/java/com/ecommerce/Entity/Products.java

```diff
@@ -5,15 +5,21 @@
 import lombok.RequiredArgsConstructor;
 import lombok.Setter;
 
+import java.util.ArrayList;
+import java.util.List;
+
 @Entity
 @Table(name = "Products")
 @RequiredArgsConstructor @Getter @Setter
-public class Products{
+public class Products extends ShoppingCart {
```

Comment:
why do your Products extend the shopping cart?  I don't see a need for this

Reply:
Updated, please see commit 681fcd0